amlb/resources.py

```diff
@@ -72,9 +73,21 @@ def git(cmd, defval=None):
                 return defval
 
         na = "NA"
-        version = git("--version")
-        is_git_repo = version and git("rev-parse --git-dir 2> /dev/null")
-        if is_git_repo:
+        git_version = git("--version")
+        if git_version is None:
+            warnings.warn(
+                "Cannot identify git version. Is git in path?",
+                RuntimeWarning
+            )
+
+        is_repo = git("rev-parse") is not None
+        if not is_repo:
+            warnings.warn(
+                "Directory '{}' is not a git repository.".format(os.getcwd()),
+                RuntimeWarning
+            )
```

Comment:
I had results without git versions, which made me think it would be useful to get a message about why that was (it was actually the `2>/dev/null` failing). But I forgot about the docker mode when writing this. I don't mind a single extra log line in docker personally (but will remove it if you feel strongly about it).

Reply:
`git` is not a requirement to run the app locally, and its absence doesn't prevent any single feature from working at this level, so I would not emit any warning here.
I consider both error and warning levels as "call to action" levels: call to fix, or call to inspect/confirm.
Here, everything is normal.

Reply:
Fair enough.